tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/subbuilder/BoolBuilder.scala

```diff
@@ -29,34 +29,18 @@ trait BoolBuilder extends UnsafeBoolBuilder {
 
   /** \A x \in set: p */
   def forall(x: TBuilderInstruction, set: TBuilderInstruction, p: TBuilderInstruction): TBuilderInstruction =
-    for {
```

Comment:
This is a nice cleanup, even without the added utility of checking for shadowing! :)